docs/user-guide/configuring.md

```diff
@@ -96,6 +96,7 @@ An environment defines global variables that are predefined. The available envir
 
 * `browser` - browser global variables.
 * `node` - Node.js global variables and Node.js scoping.
+* `commonjs` - CommonJS global variables and CommonJS scoping (use this for browser-only code that uses Browserify/WebPack).
```

Comment:
For Browserify, I would actually promote the use of "node", because Browserify brings all (? or most?) modules that exist in Node to the browser and really tries to unify both.

CommonJS is useful for webpack, Node (which offers another bunch of globals, so not so useful as an "env"), component.io (which is a dead technology imho, but still has users), and probably others (which I'm not familiar enough with). But at least CommonJS is [a standard](http://wiki.commonjs.org/wiki/Modules/1.1#Module_Context) that makes sense as an "env", given that "amd" is also in the list.